pyvium/errors/device_busy.py

```diff
@@ -0,0 +1,5 @@
+class DeviceBusyError(Exception):
+    '''Raised when the IviumSoft software all devices are busy'''
+    def __init__(self, message = "There is no available device. Please, connect one."):
```

Comment:
En el message pondria algo del estilo:
The selected device is busy, connect/select other device or wait till available

Luego podemos hacer una funcion que con un while que espere que termine para ejecutarle el proximo codigo